background/score/activities.go

```diff
@@ -162,6 +163,12 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 			return nil, err
 		}
 
+		if time.Since(profile.LastNudge[schema.BehaviorOnSymptomSpikeNudge]) > 90*time.Minute { // 90 minutes of delay between nudges
+			if profile.Metric.SymptomDelta <= 110 && metric.SymptomDelta > 110 { // from a non-spike area to a spike area
+				remindGoodBehavior = true
+			}
+		}
```

Comment:
if yesterday is 0, today > 0  then the delta = 100, will we consider a spike?

Reply:
For my understanding, `profile.Metric.SymptomDelta == 100` means the count is the same with yesterday. 110 means there are 10% of increasing for the symptoms.

For this story, it only considers the current delta (from today to yesterday) instead of coun. It checks the delta values between moving and submit notification.

Reply:
if yesterday =1 , today =3 ->  200 %,   if yesterday =0 , today = 2 -> 100%
This is the special case in behavior and symptom

Reply:
I think it is not special. It is regular. That's how we defined it.

Reply:
Another question, at this point the profile.Metrics has updated with metric , why are they different?

Reply:
I may not get your point.... if yesterday is 0, then delta will be never greater than 100 then there is no notification today?

Reply:
I want to push back to question `if yesterday is 0, today > 0 then the delta = 100, will we consider a spike?` back. The spike comes from the delta. It would be a question how we design the delta which is not included in the PR.

Reply:
I fetch profile it before I update it.

Reply:
100 will be notified. I update it to 10% now.